.github/workflows/ci.yml

```diff
@@ -101,6 +101,39 @@ jobs:
           suite: ${{ matrix.suite }}
           os: ${{ matrix.os }}
 
+  openssl_test:
+    needs: [mdl, yamllint, delivery]
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        os:
+          - "debian-9"
+          - "debian-10"
+          - "ubuntu-2004"
+          - "ubuntu-2104"
+          - "amazonlinux-2"
+          - "centos-7"
+          - "centos-8"
+          - "centos-stream-8"
+          - "fedora-latest"
+        suite:
+          - "source-openssl"
```

Comment:
Instead of doing this here, it might make sense to just add the suite to the `dokken` job above.